README.md

```diff
@@ -51,6 +51,13 @@ kubescape scan --submit --enable-host-scan --verbose
 
 </br>
 
+## Architecture in short
+### CLI
+<img src="docs/ks-cli-arch.png" width="300">
+
+### Operator
+<img src="docs/ks-operator-arch.png" width="300">
```

Comment:
I think would be nicer to add the diagrams as mermaid instead...

Reply:
I agree, Ben insisted on using an image